main/ui/src/main/java/org/cryptomator/ui/controllers/MainController.java

```diff
@@ -216,7 +216,27 @@ public void initStage(Stage stage) {
 
 	private void gracefulShutdown() {
 		vaults.filtered(Vault.NOT_LOCKED).forEach(Vault::prepareForShutdown);
-		Platform.runLater(Platform::exit);
+		if (!vaults.filtered(Vault.NOT_LOCKED).isEmpty()) {
+			ButtonType tryAgainButtonType = new ButtonType(
+					localization.getString("main.gracefulShutdown.button.tryAgain"));
+			ButtonType forceShutdownButtonType = new ButtonType(
```

Comment:
not sure, but I think you can configure a buttontype to be a "default button"? if so, I would make "force shutdown" the default option.

Reply:
I've updated the code so that 'Force shutdown' is now the default option. I did originally have 'Try again' as the default option, so the user never accidentally shuts down Cryptomator and loses data if they fat-finger their keyboard on the alert dialog. I will leave it up to you if this is better.